ank/src/cli_commands/get_workloads.rs

```diff
@@ -56,18 +59,89 @@ impl CliCommands {
 
         output_debug!("The table after filtering:\n{:?}", workload_infos);
 
-        // [impl->swdd~cli-shall-present-list-of-workloads~1]
-        let table_rows: Vec<WorkloadTableRow> = workload_infos.into_iter().map(|x| x.1).collect();
+        let mut workloads_table_data: BTreeMap<String, WorkloadTableRow> = workload_infos
+        .into_iter()
+        .map(|(i_name, row)| (i_name.to_string(), row))
+        .collect();
+
+
+        if !watch {
+            // // [impl->swdd~cli-shall-present-list-of-workloads~1]
```

Comment:
One comment string is enough.
```suggestion
            // [impl->swdd~cli-shall-present-list-of-workloads~1]
```

Reply:
Resolved in [b4d3ddc](https://github.com/eclipse-ankaios/ankaios/pull/463/commits/b4d3ddc21153780c6fd550eb20aa5408350afe06)